third_party/opa/data-and-policies/user-created-policy-3/sample_policies.rego

```diff
@@ -0,0 +1,74 @@
+package dataapi.authz
+
+verdict[output] {
+	count(rule) == 0
+	output = {"action": {"name":"Deny", "Deny": {}}, "policy": "Deny by default"}
+}
+
+verdict[outputFormatted] {
+	count(rule) > 0
+	output = rule[_]
+	actionName := output.action.name
+	actionWithoutName := json.remove(output.action, ["name"])
+	outputWithoutAction := json.remove(output, ["action"])
+	actionFormatted := {"name": actionName, output.action.name: actionWithoutName}
+	outputFormatted := object.union({"action": actionFormatted}, outputWithoutAction)
+}
+
+rule[{"action": {"name":"RedactAction", "columns": column_names}, "policy": description}] {
```

Comment:
Current module descriptions do not include RedactAction. Please open an issue to change modules accordingly.

Reply:
@shlomitk1 Do you mean add RedactAction to the module taxonomy?  I assume no change is required to fybrikmodule_types.go.  Right?  Do we have a module that knows how to do redact in our samples?

Reply:
@simanadler There is a taxonomy for actions, so there is no need to duplicate. This PR updates policies to match the taxonomy. Another PR is required to modify FybrikModules to match the taxonomy as well. No change to fybtikmodule_types is required.

Reply:
Created the issue #818 @shlomitk1